### DISCUSSION TITLE
is SM tube DEAD

### DISCUSSION THREAD

**Discussion by Cootmaster** [smplayer-dev/smtube#37] [General]
is SM Tube  dead  
ALL  i get for EVERY search is 
 "Quality  Date  Duration  Video type  Safe search 
A client error occurred: Error calling GET https://www.googleapis.com/youtube/v3/search?part=id%2Csnippet&maxResults=10&order=relevance&safeSearch=none&type=video&videoDefinition=any&videoDuration=any&q=elton+john&regionCode=US&key=<KEY>: (403) YouTube Data API v3 has not been used in project 861556708454 before or it is disabled. Enable it by visiting https://console.developers.google.com/apis/api/youtube.googleapis.com/overview?project=861556708454 then retry. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry.
No results found
Now you can fix the "Daily Limit Exceeded" problem by using a personal Youtube API key. Click here for more info.
Alternatively you can see the search results at YouTube.com."

I know about the key exceeding thing  
and to use YOU tube
but now i CANT SEARCH FOR ANYTHING
i also know  SMtube is obsolete  
so is it even working anymore
or will there be a new software to dl things

**Comment by Cootmaster** [comment]
![image18](https://github.com/user-attachments/assets/8a0ddb60-0ef2-4816-906b-05b63b9054e9)

**Comment by m-alice** [comment]
After war google againts yt-dlp, i think even newpipe is dead now.
